pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/Participant.java

```diff
@@ -46,6 +46,7 @@ public class Participant implements Cloneable {
             + "LEFT JOIN ddp_participant_exit ex on (p.ddp_participant_id = ex.ddp_participant_id "
             + "AND p.ddp_instance_id = ex.ddp_instance_id) WHERE realm.instance_name = ? ";
 
+    // TODO: should be an int, not a long. -DC
     @ColumnName(DBConstants.PARTICIPANT_ID)
```

Comment:
Did not want to mix the fix into this PR since it will likely include a handful of files.